assets/script.js

```diff
@@ -82,60 +84,41 @@ function startGame(){
     createQuestions();
 };
 
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf
+function getIndexFromId(id) {
+    var ids = ["buttonA", "buttonB", "buttonC", "buttonD"];
+    return ids.indexOf(id);
+}
+
+function checkAnswer(event) {
+    var userAnswer = getIndexFromId(event.target.id);
+    if (userAnswer === currentQuestion.answer) {
+        score++
+    } else {
+        timeLeft = timeLeft - 10;
+    }
+}
+
 
 // Creates questions and choices by pulling from arrays
 function createQuestions() {
-    var currentQuestion = questions[questionNumber];
+    currentQuestion = questions[questionNumber];
```

Comment:
no longer using var because `currentQuestion` was already set globally. This updates the global variable.